common/lib/dependabot/utils.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
+require "tmpdir"
 require "set"
 
 # TODO: in due course, these "registries" should live in a wrapper gem, not
 #       dependabot-core.
 module Dependabot
   module Utils
     BUMP_TMP_FILE_PREFIX = "dependabot_"
-    BUMP_TMP_DIR_PATH = "tmp"
+    BUMP_TMP_DIR_PATH = File.expand_path(Dir::Tmpname.create("tmp", "tmp") { nil })
```

Comment:
The double `"tmp"` is confusing, any way you could rename it slightly to clarify how they're different? I see one is the directory to create the other in, but it's not clear from the code which is which, so I had to lookup the signature which is annoying...

Reply:
I can try, but I can't do any anything against `Dir::Tmpname.create` signature being not very clear...

Reply:
No big deal, I was just thinking if you're creating `"tmp"` within `"/tmp"` maybe name the one you're creating something else? But if that has other side effects then just ignore my comment and move on.

Reply:
I'm not creating "tmp" inside "/tmp", I'm creating "tmp_<unique-identifier>" inside (relative to current dir) "tmp". Maybe not passing anything as dir prefix would make things less confusing? Or any other suggested prefix?

Reply:
Passing the first parameter empty seems to work and should be less confusing since there's no double "tmp".